deployment/apache/local_apache.conf

```diff
@@ -7,9 +7,9 @@
     SetEnv proxy-nokeepalive 1
 
     # uncomment these lines to enable apache to serve static files
-    # (replace /home/czue/src/commcare-hq with the path to your local HQ install
-    # Alias /static /home/czue/src/commcare-hq/staticfiles/
-    # <Directory /home/czue/src/commcare-hq/staticfiles>
+    # (replace /home/your_username/src/commcare-hq with the path to your local HQ install
+    # Alias /static /home/your_username/src/commcare-hq/staticfiles/
```

Comment:
we don't use apache anymore and doubt anyone is trying to keep this up to date. but suppose could be useful for someone who really wants to deploy commcare on apache